tests/Docker/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.8'
+
+services:
+  yii-runner-roadrunner:
+    container_name: yii-runner-roadrunner
+    hostname: yii-runner-roadrunner
```

Comment:
```suggestion
```

Reply:
@xepozz suggestion makes sense if the images are general ones that aren't intended to testing only.